src/google/protobuf/compiler/command_line_interface.h

```diff
@@ -353,6 +358,10 @@ class LIBPROTOC_EXPORT CommandLineInterface {
   // FileDescriptorSet should be written.  Otherwise, empty.
   string descriptor_set_name_;
 
+  // If --dependency_out was given, this is the filename to which the input file
```

Comment:
Sorry.. I should be clearer in the comment it should be "this is the path to the file where the dependency file will be written". Because it is a path. It's not just a filename.